### PR TITLE
Create Rectangles using 3 points

### PR DESCRIPTION
**Pull request by bansalra**

### CODE REVIEW

**Comment by supersonicclay** [comment]
I think we should probably factor out a lot of the `handleDraw...` functions out of the layer class into a standalone class so they can be tested. We're starting to add a lot of untested code.

File: src/lib/layers/editable-geojson-layer.js [hunk 1]
```diff
@@ -501,13 +518,19 @@ export default class EditableGeoJsonLayer extends EditableLayer {
         drawFeature = bboxPolygon([minX, minY, maxX, maxY]);
       } else if (mode === 'drawCircleFromCenter') {
         const centerCoordinates = ((selectedFeature.geometry.coordinates: any): Array<number>);
-        const radius = Math.max(distance(selectedFeature, groundCoords || center), 0.001);
+        const radius = Math.max(
+          distance(selectedFeature, groundCoords || centerCoordinates),
```
Comment (sivako) [inline src/lib/layers/editable-geojson-layer.js]: I fixed this already, You need to sync the code

File: src/lib/layers/editable-geojson-layer.js [hunk 2]
```diff
@@ -501,13 +518,19 @@ export default class EditableGeoJsonLayer extends EditableLayer {
...
+          0.001
+        );
         drawFeature = circle(centerCoordinates, radius);
       } else if (mode === 'drawCircleByBoundingBox') {
         const centerCoordinates = (selectedFeature.geometry.coordinates.map(
           (p, i) => (groundCoords && (p + groundCoords[i]) / 2) || p
         ): Array<number>);
-        const radius = Math.max(distance(selectedFeature, center || selectedFeature), 0.001);
+        const radius = Math.max(
```
Comment (sivako) [inline src/lib/layers/editable-geojson-layer.js]: same comment, need code sync

File: package.json [hunk 3]
```diff
@@ -49,13 +49,16 @@
     "@deck.gl/experimental-layers": "^6.0.3",
     "@turf/bbox": ">=4.0.0",
     "@turf/bbox-polygon": ">=4.0.0",
+    "@turf/bearing": ">=6.0.1",
     "@turf/buffer": ">=4.0.0",
     "@turf/circle": ">=4.0.0",
+    "@turf/destination": ">=6.0.1",
```
Comment (georgios-uber) [inline package.json]: `>=4.0.0`

File: src/lib/layers/editable-geojson-layer.js (hunk 2 above)
Comment (georgios-uber) [reply src/lib/layers/editable-geojson-layer.js]: Fix this.

File: src/lib/layers/editable-geojson-layer.js (hunk 1 above)
Comment (georgios-uber) [reply src/lib/layers/editable-geojson-layer.js]: Fix this.

**Comment by georgios-uber** [comment]
@supersonicclay #60

File: src/lib/layers/editable-geojson-layer.js (hunk 2 above)
Comment (bansalra) [reply src/lib/layers/editable-geojson-layer.js]: updated

File: src/lib/layers/editable-geojson-layer.js (hunk 1 above)
Comment (bansalra) [reply src/lib/layers/editable-geojson-layer.js]: Updated

**Pull request merged**